manager/src/main/java/org/apache/hertzbeat/manager/controller/AiController.java

```diff
@@ -44,6 +46,9 @@ public class AiController {
     @Autowired
     private AiServiceFactoryImpl aiServiceFactory;
 
+    /**
+     * Types of artificial intelligence
+     */
     @Value("${aiConfig.type:0}")
```

Comment:
type default is not a 0

Reply:
https://github.com/apache/hertzbeat/pull/2229：In this branch has been modified, please merge this branch first, and then I will merge type

Reply:
fix